src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -143,6 +144,13 @@ object ConductRController {
         )
       )
     )
+
+  private def toBundleName(bundle: Uri): String = {
+    val BundlePath = "(.*?)(-[a-fA-F0-9]{0,64})?\\.zip".r
```

Comment:
Instead of creating the regex every time this method is called, I'd rather move that to a val in the companion object.